services/web-server/src/servers/createApp.js

```diff
@@ -23,7 +30,27 @@ module.exports = async ({ cfg, strategies }) => {
   }).filter(o => o);
   app.use(cors({origin: allowedCORSOrigins}));
 
+  app.use(session({
+    store,
+    secret: cfg.login.sessionSecret,
+    sameSite: true,
+    resave: false,
+    saveUninitialized: false,
+    unset: 'destroy',
+    // Force a session identifier cookie to be set on every response.
+    // The expiration is reset to the original maxAge,
+    // resetting the expiration countdown.
+    rolling: true,
+    cookie: {
+      secure: url.parse(cfg.app.publicUrl).hostname !== 'localhost',
+      httpOnly: true,
+      // 15 minutes
+      maxAge: 15 * 60 * 5000,
```

Comment:
Would this mean the user is logged out of their session every 15 minutes without activity?

Reply:
Yeah. We should probably make this time longer. Perhaps 1 week?

Reply:
That makes sense.